app/views/catalog/_sort_and_per_page.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="sortAndPerPage" class="sort-pagination d-md-flex justify-content-between" role="navigation" aria-label="<%= t('blacklight.search.per_page.aria_label')%>">
+  <%= render partial: "paginate_compact", object: @response if show_pagination? %>
+  <div class="search-widgets">
+    <%= render_results_collection_tools %>
+    <div class="expand-collapse btn-group" data-controller="expandcollapse">
+      <button class="btn btn-outline-secondary active" aria-label="collapse descriptions" disabled data-action="expandcollapse#toggleDescriptions">
```

Comment:
Shouldn't these also have `aria-controls`?  If one of them always has to be selected, can this just be a styled radio button (e.g. hide the button entirely and display a image in the label)?

Reply:
Please see https://github.com/sul-dlss/earthworks/pull/1234/files to see if this is what you meant for both this and the previous comment. Also if you know the css off the top of your head for styling this I would appreciate it. What is in there now doesn't work.